packages/react-composites/src/composites/ChatComposite/selectors/baseSelectors.ts

```diff
@@ -2,8 +2,34 @@
 // Licensed under the MIT license.
 import { CommunicationIdentifierKind } from '@azure/communication-common';
 import { ChatAdapterState } from '../adapter/ChatAdapter';
+/* @conditional-compile-remove-from(stable): FILE_SHARING */
+import { FileUploadState } from '../file-sharing';
 
 /**
  * @private
  */
 export const getUserId = (state: ChatAdapterState): CommunicationIdentifierKind => state.userId;
+
+/* @conditional-compile-remove-from(stable): FILE_SHARING */
+/**
+ * @private
+ */
+export const getFileUploads = (state: ChatAdapterState): FileUploadState[] | undefined => {
+  const fileUploads = state?.fileUploads;
+  if (!fileUploads) {
+    return undefined;
+  }
+  return Object.keys(fileUploads).map((key) => fileUploads[key]);
+};
+
+/* @conditional-compile-remove-from(stable): FILE_SHARING */
+/**
+ * @private
+ */
+export const getFileUploadsCompleted = (state: ChatAdapterState): boolean | undefined => {
```

Comment:
`getFileUploads` returns an array of files, I'd expect `getFileUploadsCompleted` to return a subset of all files uploaded that are completed.

Can this be renamed to `getHaveAllFilesFinishedUploading`
Or if our selectors don't need to be named `get` just  `haveAllFilesFinishedUploading`

Reply:
Makes sense. Will change the name